### ISSUE TITLE
Plan for Uranium Fuel Cell computes Sulfuric Acid incorrectly

### ISSUE THREAD

**Issue by jlellis** [AnthorNet/SC-InteractiveMap#246]
# Issue
Computations for Sulfuric Acid are wildly incorrect.   I suspect the tool is confused because Sulfuric Acid is both an Input and a By-product.  It seems to be trying to shove ALL of the SA input through the plan rather than just the amount it needs, minus the byproduct value.  

## Replication
[1. list some replication steps
](https://satisfactory-calculator.com/en/planners/production/index/json/%7B%22Desc_NuclearFuelRod_C%22%3A%220.2%22%2C%22input%22%3A%7B%22Desc_Cement_C%22%3A%2260%22%2C%22Desc_SulfuricAcid_C%22%3A%22300%22%2C%22Desc_SteelPlateReinforced_C%22%3A%2260%22%2C%22Desc_ElectromagneticControlRod_C%22%3A%2260%22%7D%7D
)

Basically, build a plan with Cement, Sulfuric Acid, Encased Industrial Beams, and Electromagnetic control rods listed as Inputs with either mk1 Belts (60/s) or Mk1 Pipes (300m3/s).  Then look at the Graph, and the Items List.

### Actual Result
Sulfuric Acid in Items list is 304,000 m3/s.  

### Expected Result
Sulfuric Acid in Items list is either the 300m3/s I provided as an Input, or the 304m3/s it computed in the diagram....

**Comment by jlellis** [comment]
This is using the Early Access planner, not the Experimental Planner, and there are no alternate recipes enabled.
